cli.js

```diff
@@ -3,33 +3,27 @@
 const kexec = require('kexec');
 const nodeNightly = require('./');
 const existsSync = require('fs').existsSync;
-const Configstore = require('configstore');
-const pkg = require('./package.json');
-const nodeNightlyVersion = require('node-nightly-version');
+const mv = require('fs').rename;
 
-const extractDate = versionString => ~~versionString.split('nightly')[1].slice(0,8);
-const compVersion = (currentVersion, latestVersion) => extractDate(currentVersion) < extractDate(latestVersion);
 let args = process.argv.slice(2);
 
 // Check for upgrade.
 let index = args.indexOf('--upgrade');
 if(!!~index) {
-  args = args.splice(index, 1);
   nodeNightly.update().then(_ => {
     mv(`${__dirname}/node-${osArchString}`, `${__dirname}/node-nightly`);
     console.log('node-nightly is available on your CLI! ');
     process.exit(0);
-  })
+  }).catch(console.log);
```

Comment:
No it is intended to be console.log because it will print 
`You already have latest version installed.` if you want it to be error i can do it...
But i see no reason for doing that.